Donor/Controllers/DonorController.cs

```diff
@@ -44,42 +45,48 @@ public string Ping()
 		}
 
 		/// <summary>
-		/// Register a new donor
-		/// </summary>
-		/// <param name="donorDto"></param>
-		/// <returns></returns>
-		[Tags("Donor")]
-		[HttpPost("Register")]
-		public async Task<IActionResult> Register([FromBody] DonorDto donorDto)
-		{
-			
-			if (!ModelState.IsValid)
-			{
-				return BadRequest(ModelState);
-			}
-			
-			var donor = _mapper.Map<Entities.Donor>(donorDto);
-
-			await _repo.AddDonorAsync(donor);
-			await _repo.SaveChangesAsync();
-
-			return Ok("Donor registered successfully");	
-			
-		}
+        /// Register a new donor
+        /// </summary>
+        /// <param name="donorDto"></param>
+        /// <returns></returns>
+        [Tags("Donor")]
+        [HttpPost("Register")]
+        public async Task<IActionResult> Register([FromBody] DonorDto donorDto)
+        {
+            if (donorDto == null)
```

Comment:
This validation is not necessary, if the object is null the endpoint will never hit the endpoint.